### PR TITLE
Update fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md

### PR DESCRIPTION
**Pull request by PhilinTv**

### CODE REVIEW

File: 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md [hunk 1]
```diff
@@ -1,16 +1,3 @@
-> Audience:
```
Comment (PhilinTv) [inline 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md:1]: It's TW internals.
The audience should be managed by naming, portal, and navigation.
It's not a job of a reader to investigate to what auditory group they belong to.

File: 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md [hunk 2]
```diff
@@ -1,16 +1,3 @@
...
->
-> - Everyone who starts using the Docker SDK for development and production.
->
-> Outcome:
-> - You know how to choose a Docker SDK versioning approach.
-> - You know how to link your project to a particular version of the Docker SDK.
-
-
-
-This document describes why and how to select a particular version of the Docker SDK and use it in your project.
```
Comment (PhilinTv) [inline 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md:11]: If we can't set expectations with the article name, this text won't help.

File: 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md [hunk 3]
```diff
@@ -76,12 +63,16 @@ git commit -m "Added docker submodule"
 git push
 ```
 
+:::(Info) (Changing Docker SDK version)
```
Comment (PhilinTv) [inline 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md:66]: This part doesn't belong to the step-by-step of setting up the docker repo.

File: 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md [hunk 4]
```diff
@@ -57,31 +44,37 @@ Depending on your project requirements, choose one of the following ways to conf
 
 To configure git submodule:
 
-1. Create a git submodule:
+1. Using your terminal navigate to your project root folder, there you can find src/ directory and composer.json file.
```
Comment (PhilinTv) [inline 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md:47]: Submodule to be created in the project root.

File: 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md (hunk 1 above)
Comment (sprymiker) [reply 202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md:1]: That was my initiative to have this into docker/sdk repo in order to inform readers would it be useful for them or not.

I assume this is not necessary in 360 documentation.

**Comment by sprymiker** [comment]
The source of this document is in docker/sdk repository.

@andriitserkovnyi Please, manage these changes. Thanks.

**Pull request merged**